app/utils/SplitMarkdownSections.js

```diff
@@ -0,0 +1,21 @@
+import { marked } from 'marked';
+
+export function SplitMarkdownSections(markdown) {
+  const tokens = marked.lexer(markdown);
+  const sections = [];
+  let currentSection = null;
+
+  tokens.forEach(token => {
+    if (token.type === 'heading') {
+      if (currentSection) sections.push(currentSection);
```

Comment:
I realize this isn't documented anywhere (sorry), but my preference is to use brackets even for single line conditional statements like this.  I realize the code you have here is totally valid, this is just a preference I've built over the years and would like to keep consistent in this code.  Thanks!